src/Action.ts

```diff
@@ -0,0 +1,34 @@
+/* Copyright Airship and Contributors */
+
+'use strict';
+
+import { JsonValue } from './types';
+
+/**
+ * Airship Action Object.
+ * This is used to encapsulate the Action name and the Action value.
+ */
+export class Action {
```

Comment:
Instead of classes this should just be a simple interface with name/value, but we should be able to remove this